app/assets/javascripts/admin/scheduled_items.js

```diff
@@ -0,0 +1,85 @@
+Utils.fireWhenReady(['slides#edit'], function(e) {
```

Comment:
I'd like to get away from this `Utils.fireWhenReady` pattern mainly because I don't understand it. It seems like an unnecessary abstraction over jQuery's own interface. What does it do? Looking at the source itself, it's still not clicking for me.

I'd recommend a headstone comment for this entire block or file. That may be enough to clarify why `Utils.fireWhenReady` is being applied here.